### PR TITLE
pythonPackages.google*: Cleanups, updates, fix dependencies 

### PR DESCRIPTION
**Pull request by SuperSandro2000**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/google_cloud_dns/default.nix [hunk 1]
```diff
@@ -13,21 +13,29 @@ buildPythonPackage rec {
...
+
+  pythonImportsCheck = [ "google.cloud.dns" ];
+
   meta = with stdenv.lib; {
     description = "Google Cloud DNS API client library";
-    homepage = "https://github.com/GoogleCloudPlatform/google-cloud-python";
+    homepage = "https://github.com/googleapis/python-dns";
     license = licenses.asl20;
-    maintainers = [ maintainers.costrouc ];
+    maintainers = with SuperSandro2000; [ SuperSandro2000 ];
```
Comment (jonringer) [inline pkgs/development/python-modules/google_cloud_dns/default.nix:39]:
```suggestion
    maintainers = with maintainers; [ SuperSandro2000 ];
```

File: pkgs/development/python-modules/google_cloud_firestore/default.nix [hunk 2]
```diff
@@ -1,35 +1,64 @@
...
+    "google.cloud.firestore_admin_v1"
+  ];
+
   meta = with stdenv.lib; {
     description = "Google Cloud Firestore API client library";
-    homepage = "https://github.com/GoogleCloudPlatform/google-cloud-python";
+    homepage = "https://github.com/googleapis/python-firestore";
     license = licenses.asl20;
-    maintainers = [ maintainers.costrouc ];
+    maintainers = with SuperSandro2000; [ SuperSandro2000 ];
```
Comment (jonringer) [inline pkgs/development/python-modules/google_cloud_firestore/default.nix:62]:
```suggestion
    maintainers = with maintainers; [ SuperSandro2000 ];
```

File: pkgs/development/python-modules/google_cloud_language/default.nix [hunk 3]
```diff
@@ -13,20 +15,23 @@ buildPythonPackage rec {
...
+    "google.cloud.language_v1beta2"
+  ];
 
   meta = with stdenv.lib; {
     description = "Google Cloud Natural Language API client library";
-    homepage = "https://github.com/GoogleCloudPlatform/google-cloud-python";
+    homepage = "https://github.com/googleapis/python-language";
     license = licenses.asl20;
-    maintainers = [ maintainers.costrouc ];
+    maintainers = with SuperSandro2000; [ SuperSandro2000 ];
```
Comment (jonringer) [inline pkgs/development/python-modules/google_cloud_language/default.nix:35]:
```suggestion
    maintainers = with maintainers; [ SuperSandro2000 ];
```

File: pkgs/top-level/python-packages.nix [hunk 4]
```diff
@@ -2519,7 +2519,7 @@ in {
 
   google-auth-oauthlib = callPackage ../development/python-modules/google-auth-oauthlib { };
 
-  google-cloud-access-context-manager = callPackage ../development/python-modules/google-cloud-access-context-manager { };
+  google_cloud_access_context_manager = callPackage ../development/python-modules/google_cloud_access_context_manager { };
```
Comment (jonringer) [inline pkgs/top-level/python-packages.nix:2522]: You already changed a lot of these.... but 

Please normalize the attribute name. According to the [python contributing guidelines](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines), package names within nix should follow [PEP 0503](https://www.python.org/dev/peps/pep-0503/#normalized-names)

Comment (SuperSandro2000) [reply pkgs/top-level/python-packages.nix:2522]: Going to remember that for future PRs.

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 108536` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)

I think I already build all the packages locally.

**Comment by SuperSandro2000** [comment]
Going to merge this soon if there are no build failures for currently building packages so that the PR does not totally get out of hand.

File: pkgs/development/python-modules/google-cloud-texttospeech/default.nix [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    # Disable tests that require credentials
+    "test_list_voices"
+    "test_synthesize_speech"
+  ];
+
+  pythonImportsCheck = [
+    "google.cloud.texttospeach"
+    "google.cloud.texttospeach_v1"
+    "google.cloud.texttospeach_v1beta1"
+  ];
```
Comment (jonringer) [inline pkgs/development/python-modules/google-cloud-texttospeech/default.nix:35]:
```
builder for '/nix/store/k3pym2mfc5r3kcg5fs5s939m5d6hh681-python3.7-google-cloud-texttospeech-2.2.0.drv' failed with exit code 1; last 10 log lines:
  Check whether the following modules can be imported: google.cloud.texttospeach google.cloud.texttospeach_v1 google.cloud.texttospeach_v1beta1
  Traceback (most recent call last):
    File "<string>", line 1, in <module>
    File "<string>", line 1, in <lambda>
    File "/nix/store/1gc1cmg1r73629qrd7995yghq71qqar1-python3-3.7.9/lib/python3.7/importlib/__init__.py", line 127, in import_module
      return _bootstrap._gcd_import(name[level:], package, level)
    File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
    File "<frozen importlib._bootstrap>", line 983, in _find_and_load
    File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
  ModuleNotFoundError: No module named 'google.cloud.texttospeach'
```

File: pkgs/development/python-modules/google-crc32c/default.nix [hunk 6]
```diff
@@ -1,31 +1,31 @@
...
+
   propagatedBuildInputs = [ cffi ];
 
   LDFLAGS = "-L${crc32c}/lib";
   CFLAGS = "-I${crc32c}/include";
 
   checkInputs = [ pytestCheckHook crc32c ];
-  pythonImportsCheck = [ "google_crc32c" ];
+
+  pythonImportsCheck = [ "google-crc32c" ];
```
Comment (jonringer) [inline pkgs/development/python-modules/google-crc32c/default.nix:23]:
```suggestion
  pythonImportsCheck = [ "google_crc32c" ];
```

```
builder for '/nix/store/ds2pcc7kykwfz92mza17942vcbizwfqr-python3.9-google-crc32c-1.1.0.drv' failed with exit code 1; last 10 log lines:
  Check whether the following modules can be imported: google-crc32c
  Traceback (most recent call last):
    File "<string>", line 1, in <module>
    File "<string>", line 1, in <lambda>
    File "/nix/store/r5dcj0y1s34sr5vjdlvff1vgnbnvnfrp-python3-3.9.1/lib/python3.9/importlib/__init__.py", line 127, in import_module
      return _bootstrap._gcd_import(name[level:], package, level)
    File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
    File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
    File "<frozen importlib._bootstrap>", line 984, in _find_and_load_unlocked
  ModuleNotFoundError: No module named 'google-crc32c'
```

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 108536` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
2 packages built:
</summary>
<ul>
<li>python3Packages.google-cloud-texttospeech</li>
<li>python3Packages.google-crc32c</li>
</ul>
</details>

**Pull request merged**